dv/uvm/core_ibex/riscv_dv_extension/ml_testlist.yaml

```diff
@@ -65,3 +114,196 @@
+    +max_interval=100000
+    +enable_debug_seq=1
+  rtl_test: core_ibex_debug_intr_basic_test
+
+
+# --------------------------------------------------------------------------------
+# ML Parameter Constraints - riscv_rand_irq_test
+# --------------------------------------------------------------------------------
+# A description of each generation parameter can be found in the 'Configuration'
+# section of the RISC-DV documentation
+# (https://github.com/google/riscv-dv/blob/master/docs/source/configuration.rst)
+#
+# This section will provide some constraints that must be placed on the set of
+# parameters relating to simulations with external interrupts.
+# --------------------------------------------------------------------------------
+# 1)  +enable_interrupt and +require_signature_addr must both be 1.
+# 2)  If using Spike ISS, +enable_timer_irq must be 0, otherwise it can be
+#     randomized at will.
+# 3)  As before, +illegal_instr_ratio must be 0, +no_ebreak must be 1,
+#     and +no_dret must be 1.
+# 4)  +set_mstatus_tw must be 0.
+# 3)  One of the RTL simulation options +enable_irq_single_seq or
+#     +enable_irq_multiple_seq must be enabled.
+# 4)  The RTL simulation option +require_signature_addr must be 1.
+
+- test: riscv_rand_irq_test
+  description: >
+    Random test with all useful knobs
+  gen_opts: >
+    +require_signature_addr=1
+    +enable_interrupt=1
+    +enable_timer_irq=1
+    +instr_cnt=10000
+    +num_of_sub_program=5
+    +illegal_instr_ratio=0
+    +hint_instr_ratio=5
+    +no_ebreak=1
+    +no_dret=1
+    +no_wfi=0
+    +set_mstatus_tw=0
+    +no_branch_jump=0
+    +no_load_store=0
+    +no_csr_instr=0
+    +fix_sp=0
+    +enable_illegal_csr_instruction=0
+    +enable_access_invalid_csr_level=0
+    +enable_misaligned_instr=1
+    +enable_dummy_csr_write=0
+    +no_data_page=0
+    +no_directed_instr=0
+    +no_fence=0
+    +enable_unaligned_load_store=1
+    +disable_compressed_instr=0
+    +randomize_csr=1
+    +boot_mode=u
+    +stream_name_0=riscv_load_store_rand_instr_stream
+    +stream_freq_0=4
+    +stream_name_1=riscv_loop_instr
+    +stream_freq_1=4
+    +stream_name_2=riscv_hazard_instr_stream
+    +stream_freq_2=4
+    +stream_name_3=riscv_load_store_hazard_instr_stream
+    +stream_freq_3=4
+    +stream_name_4=riscv_mem_region_stress_test
+    +stream_freq_4=4
+    +stream_name_5=riscv_jal_instr
+    +stream_freq_5=4
+    +stream_name_6=riscv_int_numeric_corner_stream
+    +stream_freq_6=4
+    +stream_name_7=riscv_multi_page_load_store_instr_stream
+    +stream_freq_7=4
+    +stream_name_8=riscv_load_store_rand_addr_instr_stream
+    +stream_freq_8=4
+    +stream_name_9=riscv_single_load_store_instr_stream
+    +stream_freq_9=4
+    +stream_name_10=riscv_load_store_stress_instr_stream
+    +stream_freq_10=4
+    +dist_control_mode=1
+    +dist_shift=10
+    +dist_arithmetic=10
+    +dist_logical=10
+    +dist_compare=10
+    +dist_branch=10
+    +dist_synch=10
+    +dist_csr=10
```

Comment:
All distribution config can be removed now, they are obsolete

Reply:
Done.